aten/src/ATen/cuda/CUDAContext.h

```diff
@@ -74,6 +75,7 @@ TORCH_CUDA_CPP_API cublasHandle_t getCurrentCUDABlasHandle();
 
 #ifdef CUDART_VERSION
 TORCH_CUDA_CPP_API cusolverDnHandle_t getCurrentCUDASolverDnHandle();
+TORCH_CUDA_CPP_API cusolverSpHandle_t getCurrentCUDASolverSpHandle();
```

Comment:
I can't find the implementation for this function, probably the file was not added to the gitbranch.

Reply:
I'm sorry, I missed adding this file. It should be added now.